ui/raidboss/data/05-shb/trial/varis-ex.js

```diff
@@ -0,0 +1,103 @@
+'use strict';
+[{
+  zoneRegex: {
+    en: /^Memoria Misera \(Extreme\)$/,
+  },
+  timelineFile: 'varis-ex.txt',
+  triggers: [
+    {
+      id: 'VarisEx Phase 2',
+      regex: Regexes.startsUsing({ source: 'Varis Yae Galvus', id: '4CCC', capture: false }),
+      run: function(data) {
+        data.phase = 'phase2';
+        data.resetState();
+      },
+    },
+    {
+      id: 'VarisEx Ignis Est',
+      regex: Regexes.startsUsing({ source: 'Ignis Est', id: '4CB6', capture: false }),
+      delaySeconds: 2,
+      response: Responses.getOut(),
+    },
+    {
+      id: 'VarisEx Ventus Est',
+      regex: Regexes.startsUsing({ source: 'Ventus Est', id: '4CC7', capture: false }),
+      delaySeconds: 2,
+      response: Responses.getIn(),
```

Comment:
For mechanics that are 50/50 (Ignis/Ventus, Chariot/Dynamo, etc,) best practice is to assign one an alert and one an info, so they are sound-distinguishable. (`getIn()` and `getOut()` both default to alert.) I think most other examples in the repository have `getIn()` be info level.